.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+- '8'
+script:
+- npm test
```

Comment:
in some separate task we should fix all linter errors and add `npm lint` here as well